From: Luigi Baldoni <aloisio@gmx.com>
Date: 2018-12-01 15:40:43 +0200
Subject: Do not build native plugins

Native plugins are problematic to package on account of being
binaries being looked for into datadir. Will have to create
a better patch that looks for them inside libdir, but for now
I'm simply disabling them.

Furthermore, according to upstream, they aren't particularly useful.

---
Index: projectm-3.1.0/src/Makefile.am
===================================================================
--- projectm-3.1.0.orig/src/Makefile.am
+++ projectm-3.1.0/src/Makefile.am
@@ -10,7 +10,7 @@ if ENABLE_EMSCRIPTEN
   PROJECTM_EMSCRIPTEN_SUBDIR = projectM-emscripten
 endif
 
-SUBDIRS=libprojectM NativePresets ${PROJECTM_SDL_SUBDIR} ${PROJECTM_QT_SUBDIR} ${PROJECTM_EMSCRIPTEN_SUBDIR}
+SUBDIRS=libprojectM ${PROJECTM_SDL_SUBDIR} ${PROJECTM_QT_SUBDIR} ${PROJECTM_EMSCRIPTEN_SUBDIR}
 
 # system headers/libraries/data to install
 # for compatibility reasons here as nobase_include
